content/blog/pulumi-insights-2/index.md

```diff
@@ -15,7 +15,7 @@ Pulumi Insights 2.0 brings the power of Insights to all of your cloud infrastruc
 
 <!--more-->
 
-Today we are outlining our vision for Pulumi Insights 2.0, and the features that are part of Insights 2.0 will be released over the coming weeks and months. Sign up for the [private preview](/product/waitlist) to get early access.
+Today we are outlining our vision for Pulumi Insights 2.0, and the features that are part of Insights 2.0 will be released over the coming weeks and months. [Get started with Pulumi Insights 2.0](/docs/insights/).
```

Comment:
Should you link directly to the Get Started? https://www.pulumi.com/docs/insights/get-started/

Reply:
I think so. I pulled that from the Insights product page. Are you thinking we should link to the Insights 2.0 product page?